packages/procaptcha-react/src/components/CaptchaWidget.tsx

```diff
@@ -39,117 +38,113 @@ const getHash = (item: any) => {
+                        key={index}
+                    >
                         <Box
-                            pt={0.5}
-                            pl={0.5}
-                            sx={{
-                                // enable the items in the grid to grow in width to use up excess space
-                                flexGrow: 1,
-                                // make the width of each item 1/3rd of the width overall, i.e. 3 columns
-                                flexBasis: '33.3333%',
-                                // include the padding / margin / border in the width
-                                boxSizing: 'border-box',
-                            }}
-                            key={index}
+                            sx={{ cursor: 'pointer', height: '100%', width: '100%' }}
+                            onClick={isTouchDevice ? undefined : () => onClick(hash)}
+                            onTouchStart={isTouchDevice ? () => onClick(hash) : undefined}
                         >
-                            <Box
-                                sx={{ cursor: 'pointer', height: '100%', width: '100%' }}
-                                onClick={isTouchDevice ? undefined : () => onClick(hash)}
-                                onTouchStart={isTouchDevice ? () => onClick(hash) : undefined}
-                            >
-                                <Box sx={{ border: 1, borderColor: 'lightgray' }}>
-                                    <img
-                                        style={{
-                                            width: '100%', // image should be full width / height of the item
-                                            backgroundColor: 'lightgray', // colour of the bands when letterboxing and image
-                                            opacity: solution.includes(hash) && isTouchDevice ? '50%' : '100%', // iphone workaround
-                                            display: 'block', // removes whitespace below imgs
-                                            objectFit: 'contain', // contain the entire image in the img tag
-                                            aspectRatio: '1/1', // force AR to be 1, letterboxing images with different aspect ratios
-                                        }}
-                                        src={item.data}
-                                        alt={`Captcha image ${index + 1}`}
-                                    />
-                                </Box>
-                                <Fade in={solution.includes(hash)}>
+                            <Box sx={{ border: 1, borderColor: 'lightgray' }}>
+                                <img
+                                    style={{
+                                        width: '100%', // image should be full width / height of the item
+                                        backgroundColor: 'lightgray', // colour of the bands when letterboxing and image
```

Comment:
should this colour come from the theme?

Reply:
Potentially - this depends on how we want to handle theming in the future, so I'd prefer to hold off moving the non changing styling into the MUI theme if that's not the way we end up handling the style customisability long term